Symfony/CS/Console/Command/FixCommand.php

```diff
@@ -607,10 +568,35 @@ function (FixerInterface $a, FixerInterface $b) {
             }
         }
 
+        $rulesSets = array();
+        foreach (RulesSet::create()->getSetDefinitionNames() as $setName) {
```

Comment:
I see the current solution as not a pain, for sure won't have time to change in some way.
Moving to separate class that will only be an array in fact without *any* logic is pointless to me.